docs/README-DOCS.md

```diff
@@ -0,0 +1,44 @@
+# Documenting nbconvert
+
+[Documentation for `nbconvert`](https://nbconvert.readthedocs.org/en/latest/)
+is hosted on ReadTheDocs.
+
+## Build Documentation locally
+
+1. Change directory to documentation root:
+
+       $ cd docs
+
+2. Install requirements:
+
+       $ pip install -r requirements.txt
+
+3. Build documentation using Makefile for Linux and OS X:
+
+       $ make html
+
+   or for Windows:
+
+       $ make.bat html
+
+4. Display the documentation locally:
+
+       $ open build/html/index.html
```

Comment:
`open` doesn't do this on Linux (though I have a personal alias, because typing `xdg-open` is annoying), and I'm not sure about Windows either. I don't think there is a reliable cross platform command for this, unfortunately. Maybe it's best to give the example with e.g. `firefox`, and rely on people knowing enough to substitute that with their preferred browser if they want.